docs/source/manual/reports.rst

```diff
@@ -0,0 +1,197 @@
+   * - Open ports
+     - Shows for the identified IP addresses which ports were found to be open and thus reachable. If available the table will show the IPv4 and/or IPv6 addresses, the hostname(s) and all open ports identified on both IPv4 and IPv6 (if available). Please note that you have to manually enable IPv6 support in Dockerized environments. See the docs on how to do this.
+     - The report shows the open ports for all scanned IP addresses.
+     - The table gives an overview on the number of occurrences for each open port.
+   * - RPKI
+     - The table gives an overview of the RPKI status for the selected domain. It currently shows if RPKI is available and if the data is not expired.
+     - The table gives an overview of the RPKI status grouped per system type. It currently shows if RPKI is available and if the data is not expired.
+     - The table gives an overview on the number of occurrences of the RPKI status grouped per system type. It currently shows if RPKI is available and if the data is not expired.
+   * - Safe connections
+     - The table gives an overview of some security configurations that are recommended to be implemented to ensure safe connections (encryption). The compliance checks look at the TLS protocols and TLS Ciphers offered by the system. Each check will show whether or not the system is compliant with this security configuration. If a lack of compliance is identified, the table below will show what compliance issue was identified with what risk.
+     - The report shows per system type how many of the identified IPs/systems are compliant with the safe connections checks. If in-compliant systems are identified, a compliance issue description is provided with the corresponding risk level.
+     - The table shows the number of occurrences matching the compliance checks.
+   * - Systems
+     - The table gives an overview of which system types were identified on the system. This is performed based on the identified open ports, which can have one or more of the following labels: DICOM, DNS, Mail, Web, Other.
+     - For each identified system type all checks are grouped together.
+     - Overview of the compliance checks grouped per system type with the number of occurrences.
+   * - TLS
+     - The table shows which TLS protocol versions and TLS ciphers were identified on the system, including the status of the identified data. This means that if outdated protocols (such as SSL3) are identified, the table will show a recommendation such as ‘Phase out’.
+     - The aggregate findings of the TLS report can be found under the safe connections checks.
+     - This data is aggregated under the safe connections checks.
+   * - Vulnerability
+     - The table gives an overview of the identified CVE's on the system.
+     - The table gives an overview of the identified CVE's on the system.
+     - The table gives an overview of the identified CVE's on the system.
```

Comment:
content repeated three times?

Reply:
Not exactly, it's an table giving an overview of the differences per report for the normal, aggregate and multi report. The vulnerability report is similar for all three, thus the same explanation text is shown here.